src/js/components/ContentCard.js

```diff
@@ -0,0 +1,193 @@
+
+    if (video && video.source && activeVideo) {
+      videoMarkup = (
+        <Layer onClose={this._handleClick} closer={true} flush={true}>
+          <Video>
+            <source src={video.source} type={`video/${video.type}`}/>
+          </Video>
+        </Layer>
+      );
+    }
+
+    return videoMarkup;
+  }
+
+  render () {
+    const { thumbnail, description, heading, overline, link, onClick, video,
+      socialIcon, direction, contentPlacement, pad, className} = this.props;
+    const tileProps = Props.pick(this.props, Object.keys(Tile.propTypes));
+    delete tileProps.onClick;
+    delete tileProps.pad;
+
+    const classes = classnames(
+      CLASS_ROOT,
+      {
+        [`${CLASS_ROOT}--direction-${direction}`]: direction,
+        [`${CLASS_ROOT}--selectable`]: (link || onClick || video)
+      },
+      className
+    );
+
+    let onContentCardClick = onClick;
+    if (!onContentCardClick && (link || video)) {
+      onContentCardClick = this._handleClick;
+    }
+
+    const contentMarkup = (
+      <Box className={`${CLASS_ROOT}__content`} pad="medium">
+        <Heading tag="h5" uppercase={true} margin="none">{overline}</Heading>
+        <Heading tag="h2" strong={true}>{heading}</Heading>
+        <Paragraph margin="none">{description}</Paragraph>
+        {this._renderChildren()}
+        {(!socialIcon) ? this._renderLinkMarkup() : null}
+      </Box>
+    );
+
+    let thumbnailMarkup;
+    if (thumbnail) {
+      thumbnailMarkup = (
+        <Box className={`${CLASS_ROOT}__thumbnail`}
+          backgroundImage={`url(${thumbnail})`}
+          justify="center" align="center">
+          {(video) ? <Anchor icon={<WatchIcon size="xlarge" />} /> : null}}
+        </Box>
+      );
+    }
+
+    let first = thumbnailMarkup;
+    let second = contentMarkup;
+    let cardJustify;
+
+    if (contentPlacement === 'bottom') {
+      first = contentMarkup;
+      second = thumbnailMarkup;
+      // align thumbnail to bottom of card for bottom cardPlacement
+      cardJustify = 'between';
+    }
+
+    let cardPad = 'small';
+    let cardFull;
+    if (direction === 'row') {
+      cardPad = {vertical: 'small'};
+      cardFull = 'horizontal';
+    }
+
+    return (
+      <Tile className={classes} onClick={onContentCardClick}
+        pad={pad || cardPad} {...tileProps}>
+        <Box className="flex" direction={direction} justify={cardJustify}
+          full={cardFull} colorIndex="light-1">
+          {first}
+          {second}
+          {this._renderVideoMarkup()}
+        </Box>
+      </Tile>
+    );
+  }
+};
+
+ContentCard.propTypes = {
+  thumbnail: PropTypes.string,
+  description: PropTypes.string,
+  heading: PropTypes.string,
+  overline: PropTypes.string,
+  link: PropTypes.string,
+  linkIcon: PropTypes.element,
+  linkText: PropTypes.string,
+  video: PropTypes.shape({
+    source: PropTypes.string.isRequired,
+    type: PropTypes.string
+  }),
+  contentPlacement: PropTypes.oneOf(['top', 'bottom']),
```

Comment:
Since images, video, and text are all 'content', how about we just call this property `reverse: PropTypes.bool`? That would align with flexbox terminology and with the new Chart components.